ddl/ddl.go

```diff
@@ -1163,6 +1163,10 @@ func GetHistoryJobByID(sess sessionctx.Context, id int64) (*model.Job, error) {
 	}
 	t := meta.NewMeta(txn)
 	job, err := t.GetHistoryDDLJob(id)
+	if err != nil {
+		return nil, errors.Trace(err)
```

Comment:
1. If here return err. Do we need to roll back or cancel this txn?

Reply:
good point!